chromium-headless/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM demisto/chromium:119.0.6045.79948
+
+COPY start_headless.sh start_chrome_headless.sh /
```

Comment:
```suggestion
COPY start_headless.sh start_chrome_headless.sh 
```

Reply:
It's needed. Copying 2 fils to the root folder.